modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -3178,25 +3229,28 @@ private static Boolean canReadFromWriteIntent(UUID txId, @Nullable TransactionMe
         return false;
     }
 
-    private CompletableFuture<Void> validateAtTimestamp(UUID txId) {
+    /**
+     * Takes current timestamp and makes schema related validations at this timestamp. The current timestamp is NOT sent as safeTime
+     * timestamp.
```

Comment:
> The current timestamp is NOT sent as safeTime timestamp.

What does this comment mean and why is it important?

Reply:
Removed the confusing part